tasks/main.yml

```diff
@@ -36,3 +39,9 @@
     dest: /etc/profile.d/java_home.sh
     mode: 0644
   when: java_home is defined and java_home != ''
+
+- name: check java version
+  command: java -version
+  changed_when: false
+  register: javaversion
+- debug: var=javaversion
```

Comment:
Just checking this today—If we want to default to Java 8 on RedHat 7 / CentOS 7, then I'd rather just make it so all the default vars files that are loaded follow the same format (e.g. `{{ ansible_distribution }}-{{ ansible_distribution_version }}`).

Also, I don't want to commit debug statements in the playbook :)